users/models.py

```diff
@@ -56,6 +56,20 @@ def __str__(self):
     def __repr__(self):
         return "User(name=%s)" % (self.first_name,)
 
+    def full_name(self):
```

Comment:
This method can probably be removed, as there is a built in method `get_full_name` which is more or less equivalent to this.

Reply:
OK, will fix